app/views/layouts/_nav.html.haml

```diff
@@ -2,7 +2,7 @@
   =link_to "Menu", root_path, class: "item"
   
   - if logged_in?
-    %a.item#username Hi, #{current_user.username}!
+    %a.item#username Hi, #{current_admin.username}!
```

Comment:
Not a huge deal because it'll get resolved later, on this branch, it's still `current_user`.